xtend-contrib/src/main/java/de/oehme/xtend/contrib/Builder.xtend

```diff
@@ -54,6 +67,15 @@ class BuilderProcessor extends AbstractClassProcessor {
 					'''
 					primarySourceElement = cls
 				]
+				addMethod('''set«builtField.simpleName.toFirstUpper»''') [
```

Comment:
@szarnekow Thanks for pointing that out - I was confused by testing with an annotation that did generate setters. Removing that gives a compilation error as expected.